pkg/sql/catalog/systemschema/system.go

```diff
@@ -403,10 +403,12 @@ CREATE TABLE system.statement_diagnostics_requests(
 	statement_fingerprint STRING NOT NULL,
 	statement_diagnostics_id INT8,
 	requested_at TIMESTAMPTZ NOT NULL,
+	min_execution_latency INTERVAL NOT NULL DEFAULT '0',
+	expires_at TIMESTAMPTZ NOT NULL DEFAULT now() + '100 years',
 	CONSTRAINT "primary" PRIMARY KEY (id),
-	INDEX completed_idx (completed, id) STORING (statement_fingerprint),
+	INDEX completed_idx_new (completed, id) STORING (statement_fingerprint, min_execution_latency, expires_at),
```

Comment:
Would you consider naming this `completed_idx_v2` instead? Perhaps I'm thinking too far ahead but what if we need to change this index again do we name it `completed_idx_new_new`?

If you follow my suggestion then please don't forget to also rename the index created in the migration.